app/controllers/stats_controller.rb

```diff
@@ -21,41 +23,47 @@ def sellers_total
     Seller.all.count
   end
 
-  def progress_bar_totals
-    query = ActiveRecord::Base.connection.execute($april_progress_bar_query)
-
-    total = 0
-    query.each do |type|
-      total += type['sum'] unless type['sum'].nil?
-    end
-
-    total
-  end
-
-  def num_days_remaining
-    currentDate = Time.now
-    endDate = Date.new(2021, 4, 26)
-    remainingDays = (endDate.to_date - currentDate.to_date).round
-    return 0 if remainingDays <= 0
-
-    (endDate.to_date - currentDate.to_date).round
-  end
-
   def transaction_totals
     Item.all.count
   end
 
   def show(donation_totals, sellers_total, transaction_totals, gam_count)
-    donation_totals = '$%s' % ActionController::Base.helpers.number_with_precision(donation_totals, precision: 0, delimiter: ',')  # "$10,000"
+    outside_db_gam_contributions = 0
+    outside_db_fundaising_contributions = 0
+    outside_db_fundaising_foodcrawl_raised = 36, 573
+    outside_db_luc_raised = 47, 689
+
+    if ENV['GOOGLEDRIVE_SECRET']
+      data = Rails.cache.read('spreadsheet_data')
+      if data.nil?
+        data = LiveStats.pull
+        Rails.cache.write('spreadsheet_data', data, expires_in: 60.minutes)
+      end
+      data = LiveStats.pull
+      outside_db_fundaising_contributions = data['outside_db_fundaising_contributions'].to_i
+      outside_db_gam_contributions = data['outside_db_gam_contributions'].to_i
+      outside_db_fundaising_foodcrawl_raised = data['outside_db_fundaising_foodcrawl_raised']
+      outside_db_luc_raised =  data['outside_db_luc_raised']
+    end
+    # box1
+    donation_totals += outside_db_fundaising_contributions
+    donation_totals = '$%s' % ActionController::Base.helpers.number_with_precision(donation_totals, precision: 0, delimiter: ',') # "$10,000"
+
+    # box2
+    gam_count += outside_db_gam_contributions
     gam_count           = ActionController::Base.helpers.number_with_precision(gam_count, precision: 0,
                                                                                           delimiter: ',')
-    foodcrawl_raised    = '36,573'
-    transaction_totals  = ActionController::Base.helpers.number_with_precision(transaction_totals,
-                                                                               precision: 0, delimiter: ',')
-    sellers_total
-    luc_raised = '$47,689'
+    # box3
+    foodcrawl_raised    = outside_db_fundaising_foodcrawl_raised
+
+    # box4
+    total_vouchers_giftcards = ActionController::Base.helpers.number_with_precision(transaction_totals,
```

Comment:
should we do total items here -- or switch it to total payment_intents? there was some discussion of this with Ling and co, maybe we should take this to discord?